client/src/tests/relatedProductsTests.test.js

```diff
@@ -46,7 +52,33 @@ describe('RelatedProducts methods', () => {
     const instance = wrapper.instance();
     expect(typeof instance.handleAddToOutfit).toBe('function');
   })
+})
 
+let rpcwrapper;
```

Comment:
if you were having trouble seeing the state of the instances after using mount .... Try either 

- mounting the component and creating the instance in beforeEach
- or wrapping the mount in a promise like this 
`      return new Promise((resolve, reject) => {
        component = renderer.create(
          <QnAClicks>
            {allClicksProps => {
              return <QuestionsNAnswers allClicksProps={allClicksProps} {...props}/>
            }}

          </QnAClicks>
        )
        resolve(component)
      })
      .then(newComponent => {
        let questions = newComponent.toJSON().children[0].children[4].children
        expect(questions[0].props.className).toEqual('showQuestion')
        expect(questions[1].props.className).toEqual('showQuestion')
        expect(questions[2].props.className).toEqual('hide')
      })`

Reply:
Thank you, I will look into this. Was having a hard time seeing anything, and never could get state